### PR TITLE
[cu-32wfq3f] user bond positions

### PR DESCRIPTION
**Pull request by saadjhk**
## Issue
Integration of Bonds

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-32wfq3f Integrate bonds purchased by user and total purchased bonds](https://app.clickup.com/t/32wfq3f)

**Comment by saadjhk** [comment]
Depends on: [Subsquid Updates](https://github.com/ComposableFi/composable/pull/1451)

**Comment by seunlanlege** [comment]
co-opting your pr to run simnode/benchmarks. see: https://github.com/ComposableFi/composable/pull/1450#issuecomment-1225368528

**Comment by seunlanlege** [comment]
also doesn't touch the runtime

**Review by Dom-Roth — APPROVED** [review]

File: frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolRewardsPanel.tsx [hunk 1]
```diff
@@ -121,10 +117,10 @@ export const PoolRewardsPanel: React.FC<PoolDetailsProps> = ({
         />
       </Box>
 
-      <Item mt={4} mb={4} value={`$${rewardValue.toFormat()}`}>
+      <Item mt={4} mb={4} value={`$${0}`}>
         <Typography variant="h6">Your rewards</Typography>
       </Item>
-      {rewardsLeft.map(({ tokenId, value }: { tokenId: keyof typeof TOKENS, value: BigNumber}) => (
+      {[].map(({ tokenId, value }: { tokenId: keyof typeof TOKENS, value: BigNumber}) => (
```
Comment (andresmechali) [inline frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolRewardsPanel.tsx:123]: was this placeholder left by mistake?

**Review by andresmechali — APPROVED** [review]

File: frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolRewardsPanel.tsx (hunk 1 above)
Comment (saadjhk) [reply frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolRewardsPanel.tsx:123]: nope, this is wip

Comment (saadjhk) [reply frontend/apps/pablo/components/Organisms/pool/PoolDetails/PoolRewardsPanel.tsx:123]: and not related to bonds

**Pull request merged**